web/blog/tenzir-node-v4.27/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Tenzir Node v4.27: Amazon MSK IAM Integration"
```

Comment:
We can probably drop the `IAM` here

Reply:
Maybe not, you could technically send to MSK without this but you would need some other form of auth.